Common/Data/Consolidators/IdentityDataConsolidator.cs

```diff
@@ -66,5 +66,14 @@ public override void Update(T data)
         public override void Scan(DateTime currentLocalTime)
         {
         }
+
+        /// <summary>
+        /// Resets the consolidator
+        /// </summary>
+        public override void Reset()
+        {
+            _last = default(T);
```

Comment:
Could be directly null, it's the default for all posible T types.

Reply:
Hmm perhaps it was the rule for previous versions of .NET but apparently now is not the case: https://learn.microsoft.com/en-us/dotnet/csharp/misc/cs0403?f1url=%3FappId%3Droslyn%26k%3Dk(CS0403)